### PR TITLE
docs: Add new supported networks page

### PR DESCRIPTION
**Pull request by germartinez**

### CODE REVIEW

File: pages/advanced/api-new-networks.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Cost is based on an individual agreement if run by any third party (including Safe).
+- Primary cost factors:
+  - Deployment costs.
+  - Indexing.
+  - Cloud storage and monitoring services.
+
+**Timeline**
+
+- Based on individual agreement
+- Prioritisation for native integration ([app.safe.global](https://app.safe.global)) is primarily based on the network's smart account/account abstraction strategy (for example: is every user account a Safe under the hood beyond the usage of Safe as a multi-signature for treasury management?) and [additional gathered metrics](https://noteforms.com/forms/request-safe-ui-and-infra-support-4weugt).
```
Comment (tanay1337) [inline pages/advanced/api-new-networks.md:49]:
```suggestion
- Prioritisation for native integration ([app.safe.global](https://app.safe.global)) is primarily based on the network's smart account/account abstraction strategy (for example, is every user account a Safe under the hood beyond the usage of Safe as a multi-signature for treasury management?) and [additional metrics](https://noteforms.com/forms/request-safe-ui-and-infra-support-4weugt).
```

File: pages/advanced/api-new-networks.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## New Supported Chains
+
+Safe's vision is to make every Web3 account a smart account. Therefore, we prioritize teams and chains building with smart accounts, especially if they push account abstraction (for example, ERC-4337/EIP-1271 adoption) with Safe (learn more [here](https://docs.safe.global/home/4337-safe)).
+
+To have the full Safe stack running on your chain network. You need to consider each of these steps:
+
+### Deploy Safe canonical contracts
```
Comment (tanay1337) [inline pages/advanced/api-new-networks.md:7]: Should this be marked as a step (1) with the steps component?

File: pages/advanced/api-new-networks.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Pull Requests on Safe's GitHub are reviewed roughly on a two-week cadence.
+
+**EVM only**
+
+Safe's security is based on the contracts deployed in EVM-compatible networks. For the time being, we are not looking to extend beyond EVM.
+
+### Safe{Wallet} interface and Safe{Core} infrastructure
+
+**Process**
```
Comment (tanay1337) [inline pages/advanced/api-new-networks.md:28]: I find this section a bit confusing, can we reword and restructure this? Happy to chat sync to explain what I find confusing.

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: pages/advanced/api-new-networks.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
+import { Steps } from 'nextra/components'
+
+# New Supported Networks
+
+Safe's vision is to make every web3 account a smart account. Therefore, we prioritize teams and chains building with smart accounts, especially if they push account abstraction (for example, ERC-4337/EIP-1271 adoption) with Safe (learn more [here](https://docs.safe.global/home/4337-safe)).
+
+To have the full Safe stack running on your chain network, you need to consider the following steps:
+
+## Steps
```
Comment (tanay1337) [inline pages/advanced/api-new-networks.mdx:9]: Thoughts on getting rid of this heading and updating the next ones to H2 and H3 respectively?

**Review by tanay1337 — APPROVED** [review]

File: pages/advanced/api-new-networks.mdx (hunk 4 above)
Comment (germartinez) [reply pages/advanced/api-new-networks.mdx:9]: Without the Stepper? I'm not sure I understand the suggestion

Comment (tanay1337) [reply pages/advanced/api-new-networks.mdx:9]: Gotcha, I did not know about that. Could then rename the Steps heading to something descriptive like 'Add New Network' or something like that shows up with Google search?

Comment (germartinez) [reply pages/advanced/api-new-networks.mdx]: "Add new network" would be repetitive with the new title of the page

**Pull request merged**